### DISCUSSION TITLE
Tri-Fold for Customers

### DISCUSSION THREAD

**Discussion by brizbomb** [BitcoinDesign/Guide#1121] [Ideas]
The tri-fold for business is great but one for customers would also be useful. My business has been accepting Bitcoin for years but recently we have noticed customers asking "Why do you accept Bitcoin?" and most of the employees don't even know where to start. Being able to hand a tri-fold or flyer would improve the message and be more efficient.

**Comment by GBKS** [comment]
Thanks for posting. I cross-posted it in the repo for that specific project, [see here](https://github.com/pedromvpg/bitcoinforlocalbusiness/issues/20), and also mentioned it in the community Discord [here](https://discord.com/channels/903125802726596648/903126401178292234/1297815258567675926).
